spec/translation_maps/availability_map_ht_intl_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "ht_traject/ht_constants"
+
+module AvailabilitynMapHTINTL
+  TESTS = {
+    "pd" => [HathiTrust::Constants::FT], # 1
```

Comment:
Would it make sense just to put this in the translation map rather than having regexes and a test?

Reply:
I converted all the non-CC attrs to strings because that's a lot nicer, kept the CCs as a regex since it's simple, and kept the tests because they're tests.

If we can generate these at runtime from the Rights API, then that's a way to get rid of these two files as well as sidestep Traject's use of `eval` on whatever struct we swap in... but that's for later on. I'm happy these files are more readable.